akka-docs/src/main/paradox/stream/operators/apply.md

```diff
@@ -0,0 +1,22 @@
+# apply
```

Comment:
esp for ones like this the source/apply.html style would be much more understandable